### DISCUSSION TITLE
Can not store a log file when dlt-daemon program's static variable of dlt_user_init_state was  enum InitState::INIT_DONE;

### DISCUSSION THREAD

**Discussion by Omegavoilet** [COVESA/dlt-daemon#607] [General]
Once the dlt-daemon progra‘s static variable of dlt_user_init_state has been set to INIT_DONE, subsequent client programs of the dlt-daemon will be unable to create or open a new log storage file through the dlt_init_file function.

**Comment by Gagi2k** [comment]
Duplicate of https://github.com/COVESA/dlt-daemon/issues/705
